packages/block-library/src/archives/editor.scss

```diff
@@ -1,3 +1,9 @@
 ul.wp-block-archives {
 	padding-left: 2.5em;
 }
+
+.wp-block-archives-dropdown {
```

Comment:
This is not needed, the "style" will be applied in the editor by default.

Reply:
Okay :)